tests/testapp/tests/test_filter_classes.py

```diff
@@ -185,3 +185,14 @@ def test_unauthorized_user(self):
             reverse("test_book_list_unauthorized_view", args=(self.shelf_a.id,))
         )
         self.assertEqual(r.status_code, 401)
+
+    def test_presence_of_null_org_field(self):
+        operator = self._get_operator()
+        self._create_org_user(
+            user=operator, is_admin=True, organization=self._get_org('org_a')
+        )
+        token = self._obtain_auth_token(operator)
+        auth = dict(HTTP_AUTHORIZATION=f'Bearer {token}')
+        path = reverse('test_template_list') + '?format=api'
+        response = self.client.get(path, **auth)
```

Comment:
this is a lot cleaner:
```python
response = self.client.get(url, {'format': 'api'}, HTTP_AUTHORIZATION=f'Bearer {token}')
```

I noticed only now the same pattern in the rest of the file, I will open an issue shortly to clean that up later for now just clean this one please.

Reply:
Created issue: https://github.com/openwisp/openwisp-users/issues/242